simulator/casarc

```diff
@@ -8,8 +8,8 @@ measures.DE405.directory: /usr/share/casacore/data/ephemerides
 measures.line.directory: /usr/share/casacore/data/ephemerides
 measures.sources.directory: /usr/share/casacore/data/ephemerides
 measures.comet.directory: /usr/share/casacore/data/ephemerides
-measures.ierseop97.directory: /usr/share/casacore/data/geodetic
-measures.ierspredict.directory: /usr/share/casacore/data/geodetic
-measures.tai_utc.directory: /usr/share/casacore/data/geodetic
-measures.igrf.directory: /usr/share/casacore/data/geodetic
-measures.observatory.directory: /usr/share/casacore/data/geodetic
+measures.ierseop97.directory: /code/geodetic
```

Comment:
where do you check them out from? casacore-data is updated on a weekly basis from the ASTRON FTP server. Running an `apt-get update && apt-get -yq upgrade` should do the trick. We can also autorebuild the simulator container on a weekly basis.